### ISSUE TITLE
Question about the code in carefl/nflib/flows.py

### ISSUE THREAD

**Issue by Xiaojin-Lu** [piomonti/carefl#1]
HI,
It's a great work, but I got a question about the code.
In this [function](https://github.com/piomonti/carefl/blob/master/nflib/flows.py#L103), `t_base` is initialized  with 
`self.t_base = nn.Parameter(torch.randn(1, self.dim // 2), requires_grad=True) if shift_base else None`
 , but in the [forward function](https://github.com/piomonti/carefl/blob/master/nflib/flows.py#L128) and [backward function](https://github.com/piomonti/carefl/blob/master/nflib/flows.py#L147) , `t0` is initialized with  `t0 = self.s_base if self.t_base is not None else torch.zeros_like(z0)` , which means `self.t_base` is not used.
Is there any reason why `t0` is initialized with  `self.s_base` instead of `self.t_base` or just a code typo?
